Ghidra/Processors/V850E2M/data/languages/Instructions/Logic.sinc

```diff
@@ -0,0 +1,59 @@
+#####################################################
+#####		 	Logic			#####
+#####################################################
+
+
+# AND reg1, reg2 - rrrrr001010RRRRR
+:or R0004, R1115 is op0510=0x0A & R0004 & R1115
```

Comment:
I am already fixed this typo ;-)

Reply:
Thats crazy. I was 100% sure that I fixed that too :D